dist/utils.min.js

```diff
@@ -1,2 +1,2 @@
scope.tsDropDown)&&$scope.tsDropDown.forEach(function(dropDownItem){$scope.listItemTransclude&&$scope.listItemTransclude($scope.$new(),function(clone,scope){scope.item=dropDownItem;var listItem=ae(document.createElement("li"));listItem.attr("ng-class",'{"highlighted":highlightedItem==item}');var compiledListItem=$compile(listItem)(scope);compiledListItem.append(clone[0]),dropDownItem.hasOwnProperty("interactive")&&dropDownItem.interactive!==!0||(compiledListItem.on("click",function(){updateSelected(dropDownItem),$scope.tsItemClick&&$scope.tsItemClick({item:dropDownItem}),$scope.$apply(toggleDropDown)}),compiledListItem.on("mouseenter",function(){$scope.highlightedItem=scope.item,selectedIndex=$scope.tsDropDown.indexOf(scope.item),$scope.$apply()})),compiledListItem[0].style.width=(scope.tsDropDownWidth||textDisplayElement[0].offsetWidth+dropDownArrow[0].offsetWidth)+"px",dropDownUnorderedList.append(compiledListItem)})})}),$scope.$watch("tsDropDownWidth",function(newVal){if(newVal)for(var i=0;i<dropDownUnorderedList.children().length;i++){var child=dropDownUnorderedList.children()[i];child.style.width=newVal+"px"}}),$scope.highlightedItem=$scope.tsDropDown[selectedIndex],$scope.placeholderTransclude&&$scope.placeholderTransclude($scope.$new(),function(clone,scope){placeholderScope=scope,placeholderElement=clone[0],textDisplayElement.append(clone[0])}),textDisplayElement.on("click",function(){$scope.$apply(toggleDropDown)}),dropDownArrow.on("click",function(){$scope.$apply(toggleDropDown)}),$scope.$watch("tsDropDownShow",function(newVal,oldVal){void 0!==newVal&&toggleDropDown(newVal)}),ngModelCtrl&&($element.on("blur",function(){$scope.$apply(function(){$scope.dropDownOpen=!1})}),ngModelCtrl.$render=function(){updateSelected(ngModelCtrl.$viewValue||"")})}}}).directive("tsPlaceholder",function(){return{restrict:"E",require:"^tsDropDown",transclude:"element",link:function(scope,iElem,iAttr,dropDownController,transclude){dropDownController.setPlaceholder(transclude)}}}).directive("tsListItem",function(){return{restrict:"E",require:"^tsDropDown",transclude:"element",link:function(scope,iElem,iAttr,dropDownController,transclude){dropDownController.setListItem(transclude)}}}),function(module){try{module=angular.module("ts.utils")}catch(e){module=angular.module("ts.utils",[])}module.run(["$templateCache",function($templateCache){$templateCache.put("templates/tsDropDown.html",'<div class="drop-down-container">\n  <div class="selected-item-container">\n  </div><div class="arrow-container" ng-class="{\'arrow-default\':!dropDownOpen, \'arrow-open\':dropDownOpen}">\n  </div>\n  <div ng-show="dropDownOpen"\n       class="drop-down-list-container">\n    <ul>\n    </ul>\n  </div>\n</div>')}])}(),angular.module("ts.utils").directive("autoGrow",function($timeout){return{restrict:"A",link:function($scope,$element,$attrs){function grow(){$element[0].style.height=0,$element[0].style.height=$element[0].scrollHeight+"px"}$element.on("input",grow),$timeout(grow,!0)}}});
```

Comment:
You have to ask the repo admin to modify the rules in HoundCI. @hassanbazzi Is that you, Dean, or both?